app/src/main/java/com/company/strengthtracker/domain/util/DataSet.kt

```diff
@@ -11,6 +11,9 @@ class DataSet(
     var xMin = Float.MAX_VALUE
     var yMax = Float.MIN_VALUE
     var yMin = Float.MAX_VALUE
+    fun getX(): MutableList<Float>{
+        return coordinateArray[0]
+    }
```

Comment:
nit, maybe change this to a val accessor like
val xArr = coordinateArray[0]

then add yArr and replace usages of coordinateArray index getters in init { }